SerialPrograms/Source/PokemonSV/Resources/PokemonSV_FillingsCoordinates.h

```diff
@@ -0,0 +1,142 @@
+/*  Fillings Coordinates
+ *
+ *  From: https://github.com/PokemonAutomation/Arduino-Source
+ *
+ */
+
+#ifndef PokemonAutomation_PokemonSV_FillingsCoordinates_H
+#define PokemonAutomation_PokemonSV_FillingsCoordinates_H
+
+#include "CommonFramework/ImageTools/ImageBoxes.h"
+#include <vector>
+#include <map>
+
+namespace PokemonAutomation {
+namespace NintendoSwitch {
+namespace PokemonSV {
+
+class FillingsCoordinates {
```

Comment:
Instead of reconstructing this entire class (and thus re-initializing al these tables) inside a loop, can you make the class a lazy-init static?

You'll see this pattern in some of the other resource classes where the constructor is private is has a method: `ClassName::instance()` which simply returns a static instance of the class.

```
ClassName::instance(){
   static ClassName self(...);
   return self;
}
```
Then everywhere it's used, it's called as `ClassName::instance().do_something();`